dsaas-services/f8-forker.yaml

```diff
@@ -1,5 +1,5 @@
 services:
-- hash: master
+- hash: 62562f02d25e13756913c1a02bbf1916cda662b0
   name: fabric8-forker
```

Comment:
@vpavlin we might need to handle this in some special sort of way in the jenkins python script ? where its moving from master to a real hash